distro/commands/wiring_test.go

```diff
@@ -0,0 +1,34 @@
+package commands_test
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/projectriff/riff/distro/commands"
+	"github.com/spf13/cobra"
+)
+
+var _ = Describe("`riff-distro` root command", func() {
+	Context("should wire subcommands", func() {
+		var rootCommand *cobra.Command
+
+		BeforeEach(func() {
+			rootCommand = commands.DistroCreateAndWireRootCommand()
+		})
+
+		It("including `riff-distro` docs", func() {
+			errMsg := "`%s` should be wired to root command"
+			Expect(find(rootCommand, "docs")).NotTo(BeNil(), fmt.Sprintf(errMsg, "docs"))
+		})
+
+	})
+
+})
+
+func find(command *cobra.Command, names ...string) *cobra.Command {
```

Comment:
it's a copy of the other wiring test, I will define it in a single place instead